### DISCUSSION TITLE
Generating Smallest & Fastest Binary With Bun

### DISCUSSION THREAD

**Discussion by wakaztahir** [oven-sh/bun#6636] [General]
My question is that when you compile node js code using pkg or something like node-packer, The package's size is huge, Due to embedding modules (or nodejs apis)

Can Bun compile NodeJS code to executable that is fast and small ?

**Comment by vitalspace** [comment]
check https://github.com/theseyan/bkg

**Comment by lukasMega** [comment]
+1, there are still big executables in the output
